src/RequestProcessor.cs

```diff
@@ -67,6 +68,16 @@ internal StreamingMessage ProcessWorkerInitRequest(StreamingMessage request)
                 StreamingMessage.ContentOneofCase.WorkerInitResponse,
                 out StatusResult status);
 
+            // If the environment variable is set, spin up the custom named pipe server.
+            // This is typically used for debugging. It will throw a friendly exception if the
+            // pipe name is not a valid pipename.
+            string pipeName = Environment.GetEnvironmentVariable("PSWorkerCustomPipeName");
+            if (!string.IsNullOrEmpty(pipeName))
+            {
+                Console.WriteLine(string.Format(PowerShellWorkerStrings.SpecifiedCustomPipeName, pipeName));
```

Comment:
This message probably should be written out as a system log, which uses Console.WriteLine anyways.

Reply:
you want me to create a new `RpcLogger`?

Reply:
I did this ^